### DISCUSSION TITLE
Populate Future Labels for Frigate+

### DISCUSSION THREAD

**Discussion by russelltomkins** [blakeblackshear/frigate#8511] [Ideas]
Would it be possible to load future labels into the Frigate+ Interface? If "bird" is going to be a future label (fingers crossed it will) it would be great that while I'm training my own personal models and confirming the detection below isn't a person, it would be great to assign it the correct future tag of Bird in Frigate+. I process the image once and provide the negative (person) and positive (bird) for future models. The selection could be suffixed with "(Future)" so we know we won't receive them in the existing trained models.

From my limited knowledge, I assume this would also assist in building a collection of training images to adding a label to the base model once enough of us have submitted captures with the future tags?

PS. This is Fred and George, wild cockatoos in Australia who visit me each day. If Frigate was a personality detector, it nailed these two characters.
![image](https://github.com/blakeblackshear/frigate/assets/15333641/37edbf50-d27a-4f6a-84ce-6476bbccde71)

**Comment by blakeblackshear** [comment]
I will be looking at adding additional labels after I finish updating the base model.

**Reply by russelltomkins** [reply]
Not pressuring in any way, but any ideas on timeframes, days, weeks? If it's going to be soon, I'll hold off my batch processing until then.

**Reply by blakeblackshear** [reply]
More like months. I will be looking at adding some helpful UI features at the same time that should make it easier to go back and add labels.
